lib/config/fileconf.go

```diff
@@ -2017,6 +2017,9 @@ type Proxy struct {
 	// client connections.
 	MySQLPublicAddr apiutils.Strings `yaml:"mysql_public_addr,omitempty"`
 
+	// MySQLEngineVersion allow to overwrite proxy default mysql engine version reported by Teleport proxy.
+	MySQLEngineVersion string `yaml:"mysql_engine_version,omitempty"`
```

Comment:
Very much a nitpick, but IMO "engine" spills out implementation details. How about we call it `mysql_server_version`?

Reply:
My main motivation to use  MySQL Engine Version was  AWS docs  where Engine keyword is often used in terms of MySQL Server Version: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html

But I don't have strong preference so I will update this.